packages/lpa-questionnaire-web-app/package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav-rest-client-rest-client": "^1.0.0",
```

Comment:
Looks too long? Should be pins-clamav-rest-client?

Reply:
Done